### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by fajardargombez** [Hootrix/keyword_alert_bot#89]
**问题描述 Describe the bug**
I can't subscribe because the bot says "You have joined too many channels/supergroups"

**订阅信息 command Info**
请提供相关命令和信息，例如：
/subscribe samsung https://t.me/SicodePromo
![image_2024-04-17_12-18-05](https://github.com/Hootrix/keyword_alert_bot/assets/121300578/d4c6bcba-5e1d-4c81-9442-973b86997917)

**Comment by Hootrix** [comment]
```
Channel error, unable to use: You have joined too many channels/supergroups (caused by ImportChatInviteRequest)
```

The limitations of tg listening. You can deploy it yourself. My account is a regular account and I can only listen to 500 groups/channels max.

tg监听的限制。你可以自行部署。我的账号是普通账户  最多只能监听 500 个群组/频道

**Comment by Hootrix** [comment]
merged into #85
